sdcm/commit_log_check_thread.py

```diff
@@ -176,13 +178,26 @@ def run_thread(self):
                 message=f"CommitLogCheckThread failed: {exc.__repr__()} with traceback {trace}").publish()
         self.log.debug("CommitLogCheckThread finished")
 
+    @staticmethod
+    def get_commit_log_directory_max_exceed_time(prometheus_response):
+        # get max time diff from response per shard(line in response)
```

Comment:
I would recommend putting an example of the response in the docstring, so one can understand the logic below

Reply:
added